instrumentation/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+"""
+This library contains all android_binary targets to each test suite in instrumentation module.
+Note that:
+ - All the android_binaries are named similar to the respective class name of the test suite.
+"""
+
+android_binary(
+    name = "ExplorationPlayerTest",
```

Comment:
Should this be something like ``ExplorationPlayerTestBinary``, instead? I'd expect the instrumentation test target to be ExplorationPlayerTest since that's the one we pass to ``bazel test``

Reply:
Done